app/helpers/comments_helper.rb

```diff
@@ -53,11 +53,11 @@ def comment_count(view)
   def like_button_text(comment)
     case comment.public_reactions_count
     when 0
-      "Like"
+      I18n.t("core.like")
     when 1
-      "&nbsp;like"
+      "&nbsp;#{I18n.t('core.like').downcase}"
```

Comment:
I don't think these lines should share the same string. The top one is an imperative verb, and the others are nouns, singular and plural. I guess it doesn't work in French either, is it?